src/components/StringFormatter/StringFormatter.js

```diff
@@ -29,7 +29,7 @@ const StringFormatter = ({
       })}
       style={{
         maxWidth: width,
-        WebkitLineClamp: lines,
+        lineClamp: lines,
```

Comment:
React Testing Library would not render `-webkit-line-clamp` in the style tag, at all. 🤔 
And anyway, `line-clamp` has great support: https://caniuse.com/#search=line-clamp